### ISSUE TITLE
data-option doesn't work

### ISSUE THREAD

**Issue by IanQuentin** [paypal/JavaScriptButtons#136]
The data-option in the example doesn't work when dropped in the Add To Cart example.

data-option0="Color=Blue:8.00,Green:12.00,Red:10.00"

**Comment by GurudayalKhalsa** [comment]
+1. @IanQuentin have you come up with a workaround?

**Comment by IanQuentin** [comment]
Yes.  I made multiple buttons in div tags, a drop down box and a javascript function to hide/show the div tags as appropriate.  Not perfect but it'll work.
